nilearn/plotting/img_plotting.py

```diff
@@ -1086,6 +1087,10 @@ def plot_glass_brain(stat_map_img,
             or from vmin to vmax. Setting to 'auto' will select the latter if
             the range of the whole image is either positive or negative.
             Note: The colormap will always be set to range from -vmax to vmax.
+        resampling_interpolation : str
+            Interpolation to use when resampling the image to the destination
+            space. Can be "continuous" (default) to use 3rd-order spline
+            interpolation, or "nearest" to use nearest-neighbor mapping.
```

Comment:
It would be good here to add a sentence saying that nearest is faster, though it may be noisier in some cases.